resources/CICD/GithubActions/deploy_AppStore.yml

```diff
@@ -0,0 +1,78 @@
+name: deploy_AppStore
```

Comment:
Please naming `deploy_appstore`, also for the file

Reply:
@blyscuit Please update the name of the file, it's should be `deploy_appstore`.
And the value of this name attribute should be `Deploy Build To App Store`